libs/custom-scanner/src/custom_a4_scanner.ts

```diff
@@ -569,7 +568,7 @@ export class CustomA4Scanner implements CustomScanner {
   /**
    * Resets the hardware.
    */
-  async resetHardware(): Promise<Result<void, ErrorCode>> {
+  resetHardware(): Promise<Result<void, ErrorCode>> {
```

Comment:
I'm not sure why these functions were declared as `async` when they don't have any `await` expressions. Our linter prohibits this so I went ahead and removed the `async` declarations.